en/privacy/cookie-settings.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/privacy/websites/cookie-settings/
+
+cookie-settings-page-title = Cookie settings
+cookie-settings-page-desc = This page describes the different types of Cookies that { -brand-name-mozilla } may use, and gives you control over which types of data you agree to { -brand-name-mozilla } collecting.
+cookie-settings-breadcrumb-link = Previous page
+cookie-settings-explainer = Cookies are small files containing pieces of information that are saved to your computer or device when you visit a website. { -brand-name-mozilla } uses Cookies to help make our websites work, as well as to collect information on how you use and interact with our websites, such as the pages you visit.
+cookie-settings-page-intro = This page describes the different types of Cookies and similar technologies (such as pixel tags, web beacons, clear GIFs, JavaScript, and local storage) (“Cookies”) that { -brand-name-mozilla } may use, and gives you control over which types of data you agree to { -brand-name-mozilla } collecting.
```

Comment:
The reference to `(“Cookies”)` seems at an odd place in this sentence. 

It either comes right after `... different types of Cookies ...` or not needed.

Reply:
This is copy directly from legal, so I’m not going to change how it is worded here.